### ISSUE TITLE
BUg*** Select Overshoot to the top if options > 10

### ISSUE THREAD

**Issue by masterpowers** [Dogfalo/materialize#2298]
It will overshoot upward like this , you cant see the options since it is above the screen
dont know how to fixed this to make it behave as usual normal dropdown
then you can scroll of a long list of option dynamically generated

BTW im using the CDN of latest materialized css
```
{!! HTML::style('https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.2/css/materialize.min.css') !!}
```

Image of overshoot to the screen for iphone 6 same effect happens for a desktop client
![select overshoot](https://cloud.githubusercontent.com/assets/11954930/11167548/770d0fc6-8ba1-11e5-8d15-bab79dce7466.png)

Anyone Also Experiencing like this?
It actually works fine if your option are <10 if it is greater than 10

This is my HTML
```
<div class="row">
 <div class="input-field col s12">
    <select id="powerselect" name="sponsorlink">
    <option value="" disabled selected>Choose Sponsor Link</option>  
     
    </select>
    <label>Money Links</label>
  </div>
  </div>
```
Here is my Javascript to populate dynamic options
```
// initiate links object
                        var links = data.userdata.links
                        // remove any existing option in select / reset back 
                        $('select').children().remove();
                        // destroy 
                        $('select').material_select('destroy');
                        // iterate thru all links
                        for (var i = 0; i < links.length; i++) {
                          // append all links in options
                          $("#powerselect").append('<option value="' + links[i].link + '">' + links[i].link  + '</option>');
                          // log all links
                          console.log(links[i].link);
                          
                        };
                        // re initialize the new value for options
                        $('select').material_select();
```

**Comment by masterpowers** [comment]
![Uploading 10.png…]()
![Uploading 11.png…]()


![Uploading 10.png…]()
![Uploading 11.png…]()

**Comment by acburst** [comment]
Fixed in b18d753. Now I check if flowing upwards also goes offscreen. If that is true then I just crop the height of the dropdown to fit to the bottom of the screen.
